scan/src/components/modal/submitTx/PreviewJsonStep.re

```diff
@@ -119,7 +119,8 @@ let make = (~rawTx, ~onBack, ~account: AccountContext.t) => {
                         ~signature=signature |> JsBuffer.toBase64,
                         ~pubKey=account.pubKey,
                         ~tx=rawTx,
-                        ~mode="block",
+                        // TODO: Use sync for now
```

Comment:
Should say what to do next

Reply:
We use sync mode instead of block permanently.